apps/www/app/_components/tokens-comp/token-list/token-list.tsx

```diff
@@ -1,265 +1,120 @@
-      const key = path.toString().replace(/,/g, ' ').trim();
-      const tokens = acc.get(key);
+export const TokenList = () => {
+  const [value, setValue] = useState<string>('');
 
-      !tokens ? acc.set(key, [token]) : acc.set(key, [...tokens, token]);
+  const debouncedCallback = useDebounceCallback((value: string) => {
+    setValue(value);
+  }, 1000);
 
-      return acc;
-    }, new Map<string, Token[]>()),
+  const filteredColorTokens = Object.entries(
+    filteredRecord(colorTokens, value),
   );
-
-export const TokenList = ({
-  showThemePicker,
-  showModeSwitcher,
-  type = 'color',
-  hideValue = false,
-}: TokenListProps) => {
-  const [brand] = useState<BrandType>('digdir');
-  const [mode, setMode] = useState<'light' | 'dark'>('light');
-  const [cardColumns, setCardColumns] = useState<CardColumnType>(3);
-
-  const tokens = mode === 'light' ? tokensLight : tokensDark;
-
-  useEffect(() => {
-    setCardColumns(type === 'color' ? 3 : 2);
-  }, [type]);
-
-  const brandTypeTokens = tokens[brand][type] as unknown as Token[];
-
-  const sections = Array.from(
-    brandTypeTokens.reduce((acc, token) => {
-      const [section] = token.path;
-      acc.add(section);
-      return acc;
-    }, new Set<string>()),
+  const filteredTypographyTokens = Object.entries(
+    filteredRecord(typographyTokens, value),
   );
-
-  const sectionedTokens = Array.from(
-    sections.map((section) => [
-      section,
-      mapTokens(brandTypeTokens.filter((token) => token.path[0] === section)),
-    ]),
+  const filteredSemanticTokens = semanticTokens.filter((token) =>
+    tokenSearchFilter(token, value),
   );
 
   return (
-    <div className={classes.tokens}>
-      {(showThemePicker || showModeSwitcher) && (
-        <div className={classes.toggleGroup}>
-          {showModeSwitcher && (
-            <Dropdown.TriggerContext>
-              <Dropdown.Trigger variant='secondary'>
-                Mode: {capitalizeString(mode)}
-              </Dropdown.Trigger>
-              <Dropdown>
-                <Dropdown.List>
-                  <Dropdown.Item>
-                    <Dropdown.Button onClick={() => setMode('light')}>
-                      Light
-                    </Dropdown.Button>
-                  </Dropdown.Item>
-                  <Dropdown.Item>
-                    <Dropdown.Button onClick={() => setMode('dark')}>
-                      Dark
-                    </Dropdown.Button>
-                  </Dropdown.Item>
-                </Dropdown.List>
-              </Dropdown>
-            </Dropdown.TriggerContext>
-          )}
-        </div>
-      )}
+    <>
+      <Field>
+        <Label>Søk i design tokens</Label>
+        <Search>
+          <Search.Input
+            aria-label='Søk på variabel navn i CSS for design tokens'
+            onChange={(e) => debouncedCallback(e.target.value)}
+          />
+          <Search.Clear />
+          <Search.Button />
```

Comment:
```suggestion
```
Since we do live search, we can omit this button.
Might be relevant for our docs 🤔